packages/patternfly-4/package.json

```diff
@@ -4,33 +4,12 @@
   "config": {
     "access": "private"
   },
-  "devDependencies": {
```

Comment:
These are ignored by yarn since the package has no version, and there's nothing we can do about it :(